File: js/day02/day02.js
require('../init');
const { parseLevelReportsFile, countSafeReports } = require('./levelReports');

(function main() {
  const { levelReports } = parseLevelReportsFile({
    fileName: 'day02/input.txt',
  });
  inspect(countSafeReports({ levelReports }));
})();

File: zig/day02/day02.zig
const std = @import("std");
const lr = @import("level_reports.zig");

pub fn main() !void {
    var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
    defer arena.deinit();
    const allocator = arena.allocator();

    const lists = try lr.parseLevelReportsFile(allocator, "data/day02/input.txt");

    std.debug.print("{}\n", .{lr.safeLevelReportsCount(lists)});
}
